FluentUI.Demo/build.gradle

```diff
@@ -115,6 +116,7 @@ dependencies {
     androidTestImplementation "androidx.test.espresso:espresso-intents:${espressoVersion}"
     androidTestImplementation "androidx.test:runner:${androidxRunner}"
     androidTestImplementation "androidx.test:rules:${androidxRunner}"
+    androidTestImplementation "org.hamcrest:hamcrest:2.2"
```

Comment:
is the version supposed to be hardcoded here?

Reply:
Same as above, added this to fix some missing modules during test runs, these are present in this location only, and not intended to be used in other modules.
Do you recommend moving them out?

Reply:
lets move the version out